lusSTR/tests/test_snps.py

```diff
@@ -235,11 +235,9 @@ def test_snp_bins(tmp_path):
     lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
     all_arglist = ["snps", "all", "-w", str(tmp_path)]
     lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(all_arglist))
-    with open(obs_out, "r") as fh:
-        print(fh.read(), end="")
     assert filecmp.cmp(exp_out, obs_out) is True
-    for snp_set in range(0, 10):
-        path = tmp_path / f"evidence_samples/Kin_pos_1ng_snp_evidence_set{snp_set}.csv"
+    for snp_set in range(1, 10):
+        path = tmp_path / f"evidence_samples/Kin_pos_1ng_set{snp_set}.tsv"
```

Comment:
Changed the set # to start at 1 instead of 0 to make it more user friendly.

Changed the file extension from ```.csv``` to ```.tsv``` to also make it more user friendly (it's always been a tab-delimited file but had the .csv extension... don't ask why).